other_procore_apis/bim_web_viewer.md

```diff
@@ -832,13 +791,541 @@ Manually dispatches an event
 
 Events
 
+---
+
+## Event Names
+
+<p class="heading-link-container"><a class="heading-link" href="#event-names"></a></p>
+
+The following event names are available to notify third party code of changes in the viewer.
+
+#### Complete Example
+
+```js
+const viewer = new ProcoreBim.Webviewer(options);
+
+viewer.events.addEventListener('appResize', (data) => {
+  // Called when the window resizes and returns an object with properties
+  // `offsetHeight`, `offsetWidth`, `offsetLeft`, `offsetTop`.
+  console.log(data.offsetHeight);
+});
+
+viewer.events.addEventListener('bcfCameraSet', () => {
+  // Called when the camera has been set, does not return any data.
+});
+
+viewer.start();
+```
+
+### Callback function
+
+<p class="heading-link-container"><a class="heading-link" href="#callback-function"></a></p>
+
+The second parameter to `addEventListener` is a third party defined function to be executed when the event fires. Depending on the event, the callback function may return undefined, or data. This data may be a number, string, or a JavaScript object.
+
+### appResize
+
+<p class="heading-link-container"><a class="heading-link" href="#appresize"></a></p>
+
+Fires when the document view (window) has been resized.
+
+#### Data Properties
+
+| Field Name   | Type   | Description                                        |
+| ------------ | ------ | -------------------------------------------------- |
+| offsetHeight | number | New offset height of the webviewer parent element. |
+| offsetWidth  | number | New offset width of the webviewer parent element.  |
+| offsetLeft   | number | New offset left of the webviewer parent element.   |
+| offsetTop    | number | New offset top of the webviewer parent element.    |
+
+---
+
+### appStart
+
+<p class="heading-link-container"><a class="heading-link" href="#appstart"></a></p>
+
+Fires after .start() is called.
+
+---
+
+### bcfCameraSet
+
+<p class="heading-link-container"><a class="heading-link" href="#bcfcameraset"></a></p>
+
+Fires after a BCF Camera has been set.
+
+---
+
+### cameraUpdated
+
+<p class="heading-link-container"><a class="heading-link" href="#cameraupdated"></a></p>
+
+Fires after the camera direction or camera position has been modified.
+
+---
+
+### canvasReady
+
+<p class="heading-link-container"><a class="heading-link" href="#canvasready"></a></p>
+
+Fires after the rendering canvas has been created.
+
+---
+
+### coachmarkHyperlinkClicked
+
+<p class="heading-link-container"><a class="heading-link" href="#coachmarkhyperlinkclicked"></a></p>
+
+Fires when the coachmark indicator has been cleared. The coachmark in the upper right hand corner of the webviewer appears due to; hidden objects, applied section planes, or a field of view has been applied other than the default field of view. When you clear a coachmark, this event fires and returns a string value of which coachmark was cleared.
+
+This string can be one of the following values: `hidden`, `section`, or `fov`.
```

Comment:
I found this a little hard to parse. I tried rewriting in my native language of bullet points 🚄 . Please consider :)

```suggestion
Fires when the coachmark indicator has been cleared. The coachmark will appear in the upper right hand corner of the webviewer appear when:
- objects are hidden
- section planes are applied
- a field of view has been applied other than the default field of view

When the coachmark is cleared, this event will fire with a value indicating which type of coachmark was cleared:
- `"hidden"`
- `"section"`
- `"fov"`
```